tldr/dfg_extractor.py

```
@@ -1458,12 +1608,15 @@ def extract_ruby_dfg(code: str, function_name: str) -> DFGInfo:
 
 def _find_ruby_function_by_name(root, name: str, source: bytes):
     """Find a Ruby method node by name in tree-sitter tree."""
+
     def search(node):
```

Ruff (0.14.13)

Missing return type annotation for private function `search`

(ANN202)